package.json

```diff
@@ -1,20 +1,183 @@
 {
   "name": "linter-rubocop",
+  "main": "./lib/index.coffee",
   "version": "0.4.7",
   "description": "Lint `Ruby` on the fly, using rubocop",
   "repository": "https://github.com/AtomLinter/linter-rubocop",
   "license": "MIT",
   "engines": {
     "atom": ">=1.0.0 <2.0.0"
   },
+  "scripts": {
+    "lint": "coffeelint lib && eslint spec",
+    "test": "apm test"
+  },
+  "dependencies": {
+    "atom-linter": "^4.6.1",
+    "atom-package-deps": "^4.0.1"
+  },
+  "devDependencies": {
+    "coffeelint": "^1.15.0",
+    "eslint": "^2.9.0",
+    "eslint-config-airbnb-base": "^3.0.0",
+    "eslint-plugin-import": "^1.7.0"
+  },
+  "package-deps": [
```

Comment:
I've removed it, as adding it into the `activate` section isn't within the scope of this PR.